FabricObserver/Observers/Utilities/Telemetry/AppInsightsTelemetry.cs

```diff
@@ -310,4 +320,14 @@ public void Dispose()
             // GC.SuppressFinalize(this);
         }
     }
+
+    public enum HealthScope
```

Comment:
Why define the same enum twice with slightly different enum values?

Reply:
Please elaborate

Reply:
You also define enum HealthScope in ClusterObserver/Utilities/Telemetry/AppInsightsTelemetry.cs

Wondering why you've defined it twice

Reply:
Ah... Detritus from prototyping. Thank you... will remove duplicate...